synchronicity/synchronizer.py

```diff
@@ -317,7 +317,14 @@ async def wrapper_coro():
 
         fut = asyncio.run_coroutine_threadsafe(wrapper_coro(), loop)
         try:
-            value = fut.result()
+            while 1:
+                try:
+                    # poll every second to give Windows a chance to abort on Ctrl-C
+                    #
+                    value = fut.result(timeout=0.1)
```

Comment:
Every second or every 100ms?

Reply:
forgot to update comment when I changed the value, thanks!